.github/workflows/develop.yml

```diff
@@ -33,6 +33,8 @@ jobs:
 
       - name: "5. Install main and dev dependencies"
         run: |
+          # this avoids an unnecessary cuda install on Linux
```

Comment:
Do you think we can put it in some requirements?

Reply:
nope, see PR:

> * easy install steps for all envs
> * need fbprophet to 0.6 because they fix an easter 'egg' ([facebook/prophet#1300](https://github.com/facebook/prophet/issues/1300))
> * make sure CI uses non-cuda torch
> 
> Ideally, we would just add per-OS requirements in requirements.txt, e.g.
> 
> ```
> torch==1.6.0+cpu -f https://download.pytorch.org/whl/torch_stable.html; platform_system == "Windows"
> ```
> 
> but -f does not seem to work inside requirements.txt atm ([pypa/pip#8103](https://github.com/pypa/pip/issues/8103))